sandbox-tools/merge-repos/src/syncReposToStaging.ts

```diff
@@ -0,0 +1,778 @@
+/*
```

Comment:
Not really clear this file is the main entry point unless you read through the npm script definitions in package.json. A `bin` directory might make it more clear or `app.js` name or something.

Reply:
Yeh, I had that thought when documenting it.
Let me look to rework the support files into folders or something (lots more testing), in terms of the `bin` or `app.js` when running the script it actually compiles on the fly before running.... So there is no real `bin` instance.

Reply:
Note: I was also going to move the script from my original PR into this structure so it can reuse most of the git / github code from here.